skimage/feature/_kb.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import print_function, division
+import numpy as np
+from scipy import ndimage as ndi
+from ..util import img_as_ubyte
+from .._shared.utils import assert_nD
+
+
+def kb(image, min_scale=10, max_scale=25, saliency_threshold=0.6, clustering_threshold=7):
```

Comment:
I suggest you rename this `saliency_kadir_brady`. `kb` is definitely not informative enough!